app/models/deploy.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 class Deploy < ActiveRecord::Base
+  include AuditLog
```

Comment:
might want to include something like `AuditLog::RecordTracker` instead of the root namespace